majestic-million-feed/info.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "majestic-million-feed",
+  "label": "Majestic Million Feed",
+  "version": "1.0.0",
+  "description": "Majestic crawls the web and analyzes the data to create a huge Link Intelligence dataset describing how the world wide web links together. Use the Majestic Million connector to ingest the top known websites as 'good' indicators.",
```

Comment:
Can please add below lines at the end of the description. This is use for every feed connector.

<br/><br/>This connector has a dependency on the <a href="/content-hub/all-content/?contentType=solutionpack&amp;tag=ThreatIntelManagement" target="_blank" rel="noopener">Threat Intel Management Solution Pack</a>. Install the Solution Pack before enabling ingestion of Threat Feeds from this source.